awd/cli.py

```diff
@@ -181,25 +182,29 @@ def deposit(
                     doi,
                     e.response["Error"]["Message"],
                 )
+            # Retrieve and validate Crossref metadata
```

Comment:
I don't think it's required here, but just noting for the future, I think some whitespace is okay!  As a reader, I think I might prefer a blank line above this.  But again, just stylistic, not requested here per say (particularly since this will get another pass most likely).

Reply:
A good point!